Userland/Libraries/LibWeb/Streams/ReadableStream.cpp

```diff
@@ -93,14 +93,17 @@ WebIDL::ExceptionOr<JS::GCPtr<JS::Object>> ReadableStream::cancel(JS::Value reas
 }
 
 // https://streams.spec.whatwg.org/#rs-get-reader
-WebIDL::ExceptionOr<ReadableStreamReader> ReadableStream::get_reader()
+WebIDL::ExceptionOr<ReadableStreamReader> ReadableStream::get_reader(ReadableStreamGetReaderOptions options)
```

Comment:
This might be a lightweight struct, but no reason to copy it - pass by const ref

Reply:
Thanks @linusg , applied both comments :)